packages/rule-babel-config/src/is-valid.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * @fileoverview `babel-config-is-valid` warns again providing an invalid babel configuration file.
+ */
+import * as ajv from 'ajv';
+import * as without from 'lodash.without';
+import * as groupBy from 'lodash.groupby';
+import * as map from 'lodash.map';
+
+import { Category } from 'sonarwhal/dist/src/lib/enums/category';
+import { debug as d } from 'sonarwhal/dist/src/lib/utils/debug';
+import { IRule, RuleMetadata } from 'sonarwhal/dist/src/lib/types';
+import { RuleContext } from 'sonarwhal/dist/src/lib/rule-context';
+import { RuleScope } from 'sonarwhal/dist/src/lib/enums/rulescope';
+
+import { BabelConfigInvalid, BabelConfigInvalidSchema } from '@sonarwhal/parser-babel-config/dist/src/BabelConfigParse';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+export default class BabelConfigIsValidRule implements IRule {
+    public static readonly meta: RuleMetadata = {
+        docs: {
+            category: Category.other,
```

Comment:
We should probably add a new category `development`

Reply:
@molant do you want to have that category in this PR or should we open an issue in GitHub to add it?